mongodbatlasv2/api_access_tracking.go

```diff
@@ -71,7 +71,7 @@ type AccessTrackingApiListAccessLogsByClusterNameRequest struct {
 	start *time.Time
 }
 
-type AccessTrackingApiListAccessLogsByClusterNameQueryParams struct {
+type AccessTrackingApiListAccessLogsByClusterNameParams struct {
```

Comment:
Since we guarantee the uniqueness of the operationIds I belive that we can remove first suffix as well